### ISSUE TITLE
Report 1 Command failed: /usr/bin/env swiftlint lint --use-script-input-files --quiet --reporter json --formatError: No lintable files found at paths: ''

### ISSUE THREAD

**Issue by Qusea** [vknabel/vscode-swiftlint#58]
```
Error: Command failed: /usr/bin/env swiftlint lint --use-script-input-files --quiet --reporter json --format
Error: No lintable files found at paths: ''

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:526:28)
    at Pipe.<anonymous> (node:net:687:12)
os: darwin
```

**Comment by bode-fun** [comment]
I have this problem on Linux as well.

**Comment by vknabel** [comment]
Thanks for the report. I'll try to have a look on this soon.

**Comment by vknabel** [comment]
Should be fixed with #85 and #90
